src/custom-ui/CustomWidgets.js

```diff
@@ -3,6 +3,7 @@ import 'react-datetime/css/react-datetime.css'
 import moment from 'moment'
 import RadioWidget from '@rjsf/material-ui/lib/RadioWidget/RadioWidget'
 import CheckboxWidget from '@rjsf/material-ui/lib/CheckboxWidget/CheckboxWidget'
+import SelectWidget from '@rjsf/material-ui/lib/SelectWidget/SelectWidget'
```

Comment:
The select widget format can be fixed by using the rjsf/core package! I believe using radio and checkbox from material-ui was a custom case; the default theme used is bootstrap3 from rjsf/core.

```suggestion
import SelectWidget from '@rjsf/core/lib/components/widgets/SelectWidget'
```